builds/misc/images-release.yaml

```diff
@@ -1,354 +1,476 @@
+                      {
+                          "parameterName": "VerifyAll",
+                          "parameterValue": "/all"
+                      }
+                      ],
+                      "toolName": "sign",
+                      "toolVersion": "1.0"
+                  }
+                ]
+            signConfigType: inlineSignParams
+        - template: ../templates/dotnet3-globaljson.yaml # switch to dotnet 3 as primary install for nuget package
+        - task: DotNetCoreCLI@2
+          displayName: "Functions Binding nuget package"
+          inputs:
+            buildProperties: OutDir=$(Build.BinariesDirectory)/publish/Microsoft.Azure.WebJobs.Extensions.EdgeHub
+            command: pack
+            nobuild: true
+            packDirectory: $(Build.BinariesDirectory)/publish/
+            packagesToPack: "**/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj"
+            versionEnvVar: version
+            versioningScheme: byEnvVar
+        - template: ../templates/dotnet2-globaljson.yaml # switch to dotnet 2 as primary install for code sign
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+          displayName: "Functions Binding nuget package Sign"
+          inputs:
+            ConnectedServiceName: "Azure IoT Edge Code Sign 2"
+            FolderPath: $(Build.BinariesDirectory)/publish
+            Pattern: Microsoft.Azure.WebJobs.Extensions*.nupkg
+            inlineOperation: |
+                [
+                  {
+                      "keyCode": "CP-401405",
+                      "operationSetCode": "NuGetSign",
+                      "parameters": [],
+                      "toolName": "sign",
+                      "toolVersion": "1.0"
+                  },
+                  {
+                      "keyCode": "CP-401405",
+                      "operationSetCode": "NuGetVerify",
+                      "parameters": [],
+                      "toolName": "sign",
+                      "toolVersion": "1.0"
+                  }
+                ]
+            signConfigType: inlineSignParams
+        - bash: |
+            mkdir $(Build.ArtifactStagingDirectory)/publish-linux && \
+            mv $(Build.BinariesDirectory)/publish/{CACertificates,e2e_deployment_files,e2e_test_files,\
+            scripts,IotEdgeQuickstart*,LeafDevice*} $(Build.ArtifactStagingDirectory)/publish-linux
+          displayName: Move/stage test file artifacts
+        - task: PublishBuildArtifacts@1
+          displayName: 'Publish Artifacts to VSTS'
+          inputs:
+            PathtoPublish: '$(Build.ArtifactStagingDirectory)/publish-linux'
+            artifactName: 'publish-linux'
+        - task: PublishBuildArtifacts@1
+          displayName: Publish Dotnet Artifacts
+          inputs:
+            PathtoPublish: '$(Build.BinariesDirectory)/publish'
+            artifactName: 'dotnet_artifacts'
+            StoreAsTar: true   
+
+    - template: templates/build-broker-watchdog.yaml
```

Comment:
1. I guess the order which `broker-watchdog` is built doesn't matter. 
2. If that is the case, this can even be another stage running in parallel. (That can be done later).